lib/fizzbuzz.rb

```diff
@@ -2,7 +2,13 @@ class SuperFizzBuzz
 
   def run(input)
 
-    #Implement your code here
+    output = ""
+
+    output << "Fizz" if input % 3 == 0
+    output << "Buzz" if input % 5 == 0
+    output = input if output.empty?
+
+    output
```

Comment:
Pretty slick.

Reply:
I saw Jay use a similar method a few months back for running a FizzBuzz server with Sinatra and I was impressed with his work